generated/nidigital/nidigital/session.py

```diff
@@ -3351,12 +3939,12 @@ def write_sequencer_register(self, reg, value):
     def write_source_waveform_broadcast(self, waveform_name, waveform_data):
         r'''write_source_waveform_broadcast
 
-        TBD
+        Writes the same waveform data to all sites. Use this write method if you set the **dataMapping** parameter of the create source waveform method to Broadcast.
 
         Args:
-            waveform_name (str):
+            waveform_name (str): The name to assign to the waveform. Use the **waveformName** with source_start opcode in your pattern.
```

Comment:
waveformName --> waveform_name

(Applies in other places too)

Reply:
Done